crates/miden-testing/src/kernel_tests/tx/test_account.rs

```diff
@@ -477,25 +480,22 @@ async fn test_get_storage_slot_type() -> miette::Result<()> {
             begin
                 exec.prologue::prepare_transaction
 
-                # push the account storage item index
-                push.{item_index}
+                # push the account storage slot index
+                push.{slot_idx}
```

Comment:
indices are still used here: is this expected?
Couldn't we shift to using `get_typed_item` for getting the storage type (or otherwise refactor to not use indices?)